pkgs/tools/security/vault/default.nix

```diff
@@ -7,27 +7,32 @@ let
     rev = "e2f59b64be1fa5430fa05c91b6274284de4ea77c";
     sha256 = "10m75rp3hy71wlmnd88grmpjhqy0pwb9m8wm19l0f463xla54frd";
   };
-in buildGoPackage rec {
+in stdenv.mkDerivation rec {
```

Comment:
according to https://github.com/hashicorp/vault/blob/5d2d750d9a5771d321a918d80dd0f160941fde68/version/version_base.go#L1 you need to set `vault` into `-tags` using `buildFlags` like https://github.com/NixOS/nixpkgs/blob/bf5dc90f304b6dd047a031cd097ada3bc242b7e0/pkgs/tools/security/keybase/default.nix#L19